ci/prepare-host

```diff
@@ -5,6 +5,14 @@ set -e
 
 source $(dirname $(realpath $0))/defaults
 
+_curl() {
+	local auth_opts=""
+	if [ -n "${GITHUB_USER}" ] && [ -n "${GITHUB_TOKEN}" ]; then
+		auth_opts="-u ${GITHUB_USER}:${GITHUB_TOKEN}"
```

Comment:
You do not need user iirc. You can use the simpler -H "Authorization: token TOKEN" I think. I even used ?authorization_token= query argument in one of my projects.